src/ui/buffer.c

```diff
@@ -50,23 +50,30 @@
 #include <curses.h>
 #endif
 
+#include "log.h"
 #include "ui/window.h"
 #include "ui/buffer.h"
 
-#define BUFF_SIZE 200
+#define BUFF_SIZE           200
```

Comment:
Could you add a comment for `BUFF_SIZE` and `BUFF_SIZE_LINES` to short explain what they stand for?

Reply:
I renamed the variable to self-describing name.